fplll/wrapper.h

```diff
@@ -25,10 +25,23 @@ FPLLL_BEGIN_NAMESPACE
 /* The matrix b must not be modified before calling lll().
    lll() must be called only once. */
 
+/**
+ * @brief Wrapper. This class provides an externally callable API for LLL reducing some basis ``b``.
+ * This class forcibly instantiates some template declarations (see FPLLL_DECLARE_LLL(T) for more
+ * information at the bottom of this class) and provides an interface for calling provable,
+ * heuristic and fast variants of both LLL and HLLL.
+ *
+ * User note: all of the parameters passed into this class are non-const references. Thus, this
```

Comment:
It's not the transpose:
```
sage: from fpylll import *                                                                                                                                                                                                                    
sage: A = IntegerMatrix.random(10, "qary", k=5, q=127)                                                                                                                                                                                        
sage: U = IntegerMatrix(10, 10)                                                                                                                                                                                                               
sage: B = LLL.reduction(copy(A), U=U)                                                                                                                                                                                                         
sage: U.to_matrix(matrix(ZZ, 10, 10)) * A.to_matrix(matrix(ZZ, 10, 10)) == B.to_matrix(matrix(ZZ, 10, 10))                                                                                                                                    
True
```

We don't have matrix-matrix products in FPyLLL so I'm using Sage for that.

Reply:
I wrote ``U^T`` because we use row-representation for both ``A`` and ``U``, no? Otherwise we'd be multiplying a column by a row when we want to multiply row by row. (It's very possible I'm mis-understanding something here!)